api/routes/books/bookRoutes.ts

```diff
@@ -1,10 +1,11 @@
-const router = require('express').Router();
+const Router = require('express-promise-router');

```

Comment:
Nigdy z tego nie korzystałem, co to nam wnosi do aplikacji?

Reply:
Ułatwia tworzenie asynchronicznych zapytań do bazy